src/test/java/uk/gov/pay/api/it/GetPaymentITest.java

```diff
@@ -319,4 +319,19 @@ private ValidatableResponse getPaymentEventsResponse(String bearerToken, String
                 .get(String.format("/v1/payments/%s/events", paymentId))
                 .then();
     }
+
+    @Test
+    public void getPayment_ReturnsPaymentWithCaptureUrl() {
+        publicAuthMock.mapBearerTokenToAccountId(API_KEY, GATEWAY_ACCOUNT_ID);
+        connectorMock.respondWithChargeFoundAndCaptureUrl(250, GATEWAY_ACCOUNT_ID, CHARGE_ID, CAPTURED, RETURN_URL,
```

Comment:
It’s a bit unrealistic that the status is `CAPTURED` because such a payment shouldn’t return a capture URL. I have more thoughts on this, which I’ll leave on `ConnectorMockClient`.